packages/ui/src/theme/tokens/colors.ts

```diff
@@ -182,7 +182,7 @@ export const colors: Colors = {
     10: { value: 'hsl(210, 5%, 98%)' },
     20: { value: 'hsl(210, 5%, 94%)' },
     40: { value: 'hsl(210, 5%, 87%)' },
-    60: { value: 'hsl(210, 8%, 55%)' },
+    60: { value: 'hsl(210, 6%, 70%)' },
```

Comment:
To match the designs, making the primary border color a touch lighter. It still passes a11y color contrast: 
<img width="728" alt="CleanShot 2022-11-03 at 23 23 26@2x" src="https://user-images.githubusercontent.com/321279/199905693-510b0189-441b-4e8f-bd2d-4cb2e94f2505.png">